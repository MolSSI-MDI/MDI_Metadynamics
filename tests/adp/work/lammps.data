Untitled

22 atoms
21 bonds
36 angles
67 dihedrals
0 impropers

8 atom types
21 bond types
36 angle types
67 dihedral types

  1.4862590  71.4862590 xlo xhi
  1.0000000  71.0000000 ylo yhi
 -2.1312016  67.8687984 zlo zhi

Masses

1   1.0080000
2  12.0100000
3  12.0100000
4  16.0000000
5  14.0100000
6   1.0080000
7  12.0100000
8   1.0080000

Bond Coeffs

1 harmonic    3.40000000e+02    1.09000000e+00
2 harmonic    3.40000000e+02    1.09000000e+00
3 harmonic    3.40000000e+02    1.09000000e+00
4 harmonic    3.17000000e+02    1.52200000e+00
5 harmonic    5.70000000e+02    1.22900000e+00
6 harmonic    4.90000000e+02    1.33500000e+00
7 harmonic    4.34000000e+02    1.01000000e+00
8 harmonic    3.37000000e+02    1.44900000e+00
9 harmonic    3.40000000e+02    1.09000000e+00
10 harmonic    3.10000000e+02    1.52600000e+00
11 harmonic    3.17000000e+02    1.52200000e+00
12 harmonic    3.40000000e+02    1.09000000e+00
13 harmonic    3.40000000e+02    1.09000000e+00
14 harmonic    3.40000000e+02    1.09000000e+00
15 harmonic    5.70000000e+02    1.22900000e+00
16 harmonic    4.90000000e+02    1.33500000e+00
17 harmonic    4.34000000e+02    1.01000000e+00
18 harmonic    3.37000000e+02    1.44900000e+00
19 harmonic    3.40000000e+02    1.09000000e+00
20 harmonic    3.40000000e+02    1.09000000e+00
21 harmonic    3.40000000e+02    1.09000000e+00

Angle Coeffs

1 harmonic    3.50000000e+01    1.09500047e+02
2 harmonic    3.50000000e+01    1.09500047e+02
3 harmonic    5.00000000e+01    1.09500047e+02
4 harmonic    8.00000000e+01    1.20400052e+02
5 harmonic    7.00000000e+01    1.16600050e+02
6 harmonic    3.50000000e+01    1.09500047e+02
7 harmonic    5.00000000e+01    1.09500047e+02
8 harmonic    5.00000000e+01    1.09500047e+02
9 harmonic    5.00000000e+01    1.20000051e+02
10 harmonic    5.00000000e+01    1.21900052e+02
11 harmonic    8.00000000e+01    1.22900053e+02
12 harmonic    5.00000000e+01    1.09500047e+02
13 harmonic    8.00000000e+01    1.09700047e+02
14 harmonic    6.30000000e+01    1.10100047e+02
15 harmonic    5.00000000e+01    1.18040051e+02
16 harmonic    5.00000000e+01    1.09500047e+02
17 harmonic    5.00000000e+01    1.09500047e+02
18 harmonic    5.00000000e+01    1.09500047e+02
19 harmonic    8.00000000e+01    1.20400052e+02
20 harmonic    7.00000000e+01    1.16600050e+02
21 harmonic    5.00000000e+01    1.09500047e+02
22 harmonic    5.00000000e+01    1.09500047e+02
23 harmonic    6.30000000e+01    1.11100048e+02
24 harmonic    3.50000000e+01    1.09500047e+02
25 harmonic    3.50000000e+01    1.09500047e+02
26 harmonic    3.50000000e+01    1.09500047e+02
27 harmonic    5.00000000e+01    1.20000051e+02
28 harmonic    5.00000000e+01    1.21900052e+02
29 harmonic    8.00000000e+01    1.22900053e+02
30 harmonic    5.00000000e+01    1.09500047e+02
31 harmonic    5.00000000e+01    1.09500047e+02
32 harmonic    5.00000000e+01    1.09500047e+02
33 harmonic    5.00000000e+01    1.18040051e+02
34 harmonic    3.50000000e+01    1.09500047e+02
35 harmonic    3.50000000e+01    1.09500047e+02
36 harmonic    3.50000000e+01    1.09500047e+02

Dihedral Coeffs

1 multi/harmonic    8.00000000e-01    8.00000000e-01    0.00000000e+00    0.00000000e+00    0.00000000e+00
2 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
3 charmm    8.00000000e-02         3       180         0
4 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
5 charmm    2.50000000e+00         2       180         0
6 charmm    2.50000000e+00         2       180         0
7 charmm    1.05000000e+01         2       180         0
8 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
9 charmm    8.00000000e-02         3       180         0
10 multi/harmonic    8.00000000e-01    8.00000000e-01    0.00000000e+00    0.00000000e+00    0.00000000e+00
11 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
12 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
13 charmm    8.00000000e-02         3       180         0
14 multi/harmonic    8.00000000e-01    8.00000000e-01    0.00000000e+00    0.00000000e+00    0.00000000e+00
15 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
16 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
17 multi/harmonic    2.00000000e+00    2.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
18 multi/harmonic    0.00000000e+00    0.00000000e+00    3.60000000e+00    0.00000000e+00    0.00000000e+00
19 multi/harmonic    8.00000000e-01   -2.40000000e+00    0.00000000e+00    3.20000000e+00    0.00000000e+00
20 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
21 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
22 multi/harmonic    0.00000000e+00    0.00000000e+00    5.40000000e-01    0.00000000e+00    0.00000000e+00
23 multi/harmonic    4.20000000e-01   -1.26000000e+00    0.00000000e+00    1.68000000e+00    0.00000000e+00
24 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
25 charmm    1.10000000e+00         2       180         0
26 multi/harmonic    2.00000000e+00    2.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
27 charmm    2.50000000e+00         2       180         0
28 charmm    2.50000000e+00         2       180         0
29 multi/harmonic    1.55555545e-01   -4.66666635e-01    0.00000000e+00    6.22222180e-01    0.00000000e+00
30 multi/harmonic    1.55555545e-01   -4.66666635e-01    0.00000000e+00    6.22222180e-01    0.00000000e+00
31 multi/harmonic    1.55555545e-01   -4.66666635e-01    0.00000000e+00    6.22222180e-01    0.00000000e+00
32 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
33 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
34 charmm    4.50000000e-01         1       180         0
35 charmm    5.50000000e-01         3       180         0
36 charmm    1.58000000e+00         2       180         0
37 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
38 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
39 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
40 charmm    2.50000000e+00         2       180         0
41 charmm    2.50000000e+00         2       180         0
42 charmm    1.05000000e+01         2       180         0
43 multi/harmonic    1.55555545e-01   -4.66666635e-01    0.00000000e+00    6.22222180e-01    0.00000000e+00
44 multi/harmonic    1.55555545e-01   -4.66666635e-01    0.00000000e+00    6.22222180e-01    0.00000000e+00
45 multi/harmonic    1.55555545e-01   -4.66666635e-01    0.00000000e+00    6.22222180e-01    0.00000000e+00
46 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
47 multi/harmonic    8.00000000e-01    8.00000000e-01    0.00000000e+00    0.00000000e+00    0.00000000e+00
48 charmm    8.00000000e-02         3       180         0
49 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
50 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
51 multi/harmonic    2.00000000e-01    2.00000000e-01    0.00000000e+00    0.00000000e+00    0.00000000e+00
52 multi/harmonic    0.00000000e+00    0.00000000e+00    4.00000000e-01    0.00000000e+00    0.00000000e+00
53 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
54 multi/harmonic    4.00000000e-01   -1.20000000e+00    0.00000000e+00    1.60000000e+00    0.00000000e+00
55 multi/harmonic    1.55555545e-01   -4.66666635e-01    0.00000000e+00    6.22222180e-01    0.00000000e+00
56 multi/harmonic    1.55555545e-01   -4.66666635e-01    0.00000000e+00    6.22222180e-01    0.00000000e+00
57 multi/harmonic    1.55555545e-01   -4.66666635e-01    0.00000000e+00    6.22222180e-01    0.00000000e+00
58 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
59 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
60 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
61 charmm    1.10000000e+00         2       180         0
62 multi/harmonic    2.00000000e+00    2.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
63 charmm    2.50000000e+00         2       180         0
64 charmm    2.50000000e+00         2       180         0
65 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
66 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00
67 multi/harmonic    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00    0.00000000e+00

Atoms

     1      1      1 0.11230000    2.0000010    1.0000000   -0.0000013
     2      1      2 -0.36620000    2.0000010    2.0900000    0.0000001
     3      1      1 0.11230000    1.4862640    2.4538490    0.8898240
     4      1      1 0.11230000    1.4862590    2.4538520   -0.8898200
     5      1      3 0.59720000    3.4274200    2.6407950   -0.0000030
     6      1      4 -0.56790000    4.3905800    1.8774060   -0.0000066
     7      2      5 -0.41570000    3.5553754    3.9696488   -0.0000031
     8      2      6 0.27190000    2.7331200    4.5561601   -0.0000013
     9      2      7 0.03370000    4.8532621    4.6139253   -0.0000043
    10      2      8 0.08230000    5.4075960    4.3155388    0.8898152
    11      2      2 -0.18250000    5.6613044    4.2208425   -1.2321480
    12      2      1 0.06030000    5.1232615    4.5213630   -2.1312016
    13      2      1 0.06030000    6.6304840    4.7189354   -1.2057907
    14      2      1 0.06030000    5.8085401    3.1408724   -1.2413850
    15      2      3 0.59730000    4.7126759    6.1294185    0.0000014
    16      2      4 -0.56790000    3.6006445    6.6527027    0.0000062
    17      3      5 -0.41570000    5.8460533    6.8348833    0.0000025
    18      3      6 0.27190000    6.7370014    6.3591620   -0.0000004
    19      3      2 -0.14900000    5.8460551    8.2838837    0.0000062
    20      3      8 0.09760000    4.8185761    8.6477349    0.0000104
    21      3      8 0.09760000    6.3597984    8.6477313    0.8898283
    22      3      8 0.09760000    6.3597900    8.6477354   -0.8898187

Velocities

     1   0.0000000   0.0000000   0.0000000
     2   0.0000000   0.0000000   0.0000000
     3   0.0000000   0.0000000   0.0000000
     4   0.0000000   0.0000000   0.0000000
     5   0.0000000   0.0000000   0.0000000
     6   0.0000000   0.0000000   0.0000000
     7   0.0000000   0.0000000   0.0000000
     8   0.0000000   0.0000000   0.0000000
     9   0.0000000   0.0000000   0.0000000
    10   0.0000000   0.0000000   0.0000000
    11   0.0000000   0.0000000   0.0000000
    12   0.0000000   0.0000000   0.0000000
    13   0.0000000   0.0000000   0.0000000
    14   0.0000000   0.0000000   0.0000000
    15   0.0000000   0.0000000   0.0000000
    16   0.0000000   0.0000000   0.0000000
    17   0.0000000   0.0000000   0.0000000
    18   0.0000000   0.0000000   0.0000000
    19   0.0000000   0.0000000   0.0000000
    20   0.0000000   0.0000000   0.0000000
    21   0.0000000   0.0000000   0.0000000
    22   0.0000000   0.0000000   0.0000000

Bonds

     1      1 1 2
     2      2 2 3
     3      3 2 4
     4      4 2 5
     5      5 5 6
     6      6 5 7
     7      7 7 8
     8      8 7 9
     9      9 9 10
    10     10 9 11
    11     11 9 15
    12     12 11 12
    13     13 11 13
    14     14 11 14
    15     15 15 16
    16     16 15 17
    17     17 17 18
    18     18 17 19
    19     19 19 20
    20     20 19 21
    21     21 19 22

Angles

     1      1 1 2 3
     2      2 1 2 4
     3      3 1 2 5
     4      4 2 5 6
     5      5 2 5 7
     6      6 3 2 4
     7      7 3 2 5
     8      8 4 2 5
     9      9 5 7 8
    10     10 5 7 9
    11     11 6 5 7
    12     12 7 9 10
    13     13 7 9 11
    14     14 7 9 15
    15     15 8 7 9
    16     16 9 11 12
    17     17 9 11 13
    18     18 9 11 14
    19     19 9 15 16
    20     20 9 15 17
    21     21 10 9 11
    22     22 10 9 15
    23     23 11 9 15
    24     24 12 11 13
    25     25 12 11 14
    26     26 13 11 14
    27     27 15 17 18
    28     28 15 17 19
    29     29 16 15 17
    30     30 17 19 20
    31     31 17 19 21
    32     32 17 19 22
    33     33 18 17 19
    34     34 20 19 21
    35     35 20 19 22
    36     36 21 19 22

Dihedrals

     1      1 1 2 5 6
     2      2 1 2 5 6
     3      3 1 2 5 6
     4      4 1 2 5 7
     5      5 2 5 7 8
     6      6 2 5 7 9
     7      7 2 7 5 6
     8      8 3 2 5 6
     9      9 3 2 5 6
    10     10 3 2 5 6
    11     11 3 2 5 7
    12     12 4 2 5 6
    13     13 4 2 5 6
    14     14 4 2 5 6
    15     15 4 2 5 7
    16     16 5 7 9 10
    17     17 5 7 9 11
    18     18 5 7 9 11
    19     19 5 7 9 11
    20     20 5 7 9 11
    21     21 5 7 9 15
    22     22 5 7 9 15
    23     23 5 7 9 15
    24     24 5 7 9 15
    25     25 5 9 7 8
    26     26 6 5 7 8
    27     27 6 5 7 8
    28     28 6 5 7 9
    29     29 7 9 11 12
    30     30 7 9 11 13
    31     31 7 9 11 14
    32     32 7 9 15 16
    33     33 7 9 15 17
    34     34 7 9 15 17
    35     35 7 9 15 17
    36     36 7 9 15 17
    37     37 8 7 9 10
    38     38 8 7 9 11
    39     39 8 7 9 15
    40     40 9 15 17 18
    41     41 9 15 17 19
    42     42 9 17 15 16
    43     43 10 9 11 12
    44     44 10 9 11 13
    45     45 10 9 11 14
    46     46 10 9 15 16
    47     47 10 9 15 16
    48     48 10 9 15 16
    49     49 10 9 15 17
    50     50 11 9 15 16
    51     51 11 9 15 17
    52     52 11 9 15 17
    53     53 11 9 15 17
    54     54 11 9 15 17
    55     55 12 11 9 15
    56     56 13 11 9 15
    57     57 14 11 9 15
    58     58 15 17 19 20
    59     59 15 17 19 21
    60     60 15 17 19 22
    61     61 15 19 17 18
    62     62 16 15 17 18
    63     63 16 15 17 18
    64     64 16 15 17 19
    65     65 18 17 19 20
    66     66 18 17 19 21
    67     67 18 17 19 22
