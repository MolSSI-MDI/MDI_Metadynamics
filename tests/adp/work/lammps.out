Parsing mdi command
Parsing: -role ENGINE -name LAMMPS -method TCP -port 8021 -hostname localhost
LAMMPS (20 Apr 2018)
Reading data file ...
  orthogonal box = (1.48626 1 -2.1312) to (71.4863 71 67.8688)
  1 by 1 by 1 MPI processor grid
  reading atoms ...
  22 atoms
  reading velocities ...
  22 velocities
  scanning bonds ...
  3 = max bonds/atom
  scanning angles ...
  6 = max angles/atom
  scanning dihedrals ...
  21 = max dihedrals/atom
  reading bonds ...
  21 bonds
  reading angles ...
  36 angles
  reading dihedrals ...
  67 dihedrals
Finding 1-2 1-3 1-4 neighbors ...
  special bond factors lj:   0          0          0.5       
  special bond factors coul: 0          0          0.833333  
  4 = max # of 1-2 neighbors
  7 = max # of 1-3 neighbors
  13 = max # of 1-4 neighbors
  16 = max # of special neighbors
MDI command: <NATOMS
MDI command: EXIT
Total wall time: 0:00:00
